{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3377", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3377/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3377/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3377/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/3377", "id": 1685998073, "node_id": "I_kwDOJKSTjM5kfkn5", "number": 3377, "title": "Unknown command 'execute_python'.", "user": {"login": "iva-nova-e-katerina", "id": 15146371, "node_id": "MDQ6VXNlcjE1MTQ2Mzcx", "avatar_url": "https://avatars.githubusercontent.com/u/15146371?v=4", "gravatar_id": "", "url": "https://api.github.com/users/iva-nova-e-katerina", "html_url": "https://github.com/iva-nova-e-katerina", "followers_url": "https://api.github.com/users/iva-nova-e-katerina/followers", "following_url": "https://api.github.com/users/iva-nova-e-katerina/following{/other_user}", "gists_url": "https://api.github.com/users/iva-nova-e-katerina/gists{/gist_id}", "starred_url": "https://api.github.com/users/iva-nova-e-katerina/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/iva-nova-e-katerina/subscriptions", "organizations_url": "https://api.github.com/users/iva-nova-e-katerina/orgs", "repos_url": "https://api.github.com/users/iva-nova-e-katerina/repos", "events_url": "https://api.github.com/users/iva-nova-e-katerina/events{/privacy}", "received_events_url": "https://api.github.com/users/iva-nova-e-katerina/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 5380076298, "node_id": "LA_kwDOJKSTjM8AAAABQK1zCg", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/AI%20model%20limitation", "name": "AI model limitation", "color": "c5def5", "default": false, "description": "Not related to AutoGPT directly."}], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 3, "created_at": "2023-04-27T02:45:44Z", "updated_at": "2023-05-01T22:28:38Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### \u26a0\ufe0f Search for existing issues first \u26a0\ufe0f\n\n- [X] I have searched the existing issues, and there is no existing issue for my problem\n\n### Which Operating System are you using?\n\nLinux\n\n### Which version of Auto-GPT are you using?\n\nLatest Release\n\n### GPT-3 or GPT-4?\n\nGPT-3.5\n\n### Steps to reproduce \ud83d\udd79\n\nsudo python3 -m autogpt  --continuous --gpt3only\r\ny\n\n### Current behavior \ud83d\ude2f\n\nNEXT ACTION:  COMMAND = execute_python ARGUMENTS = {'code': \"import zipfile\\nwith zipfile.ZipFile('/home/user/Auto-GPT/auto_gpt_workspace/realestate.zip', 'r') as zip_ref:\\n zip_ref.extractall('/home/user/Auto-GPT/auto_gpt_workspace/realestate')\\n\"}\r\nSYSTEM:  Command execute_python returned: Unknown command 'execute_python'. Please refer to the 'COMMANDS' list for available commands and only respond in the specified JSON format.\r\n\n\n### Expected behavior \ud83e\udd14\n\nrun python script\n\n### Your prompt \ud83d\udcdd\n\n```yaml\r\n# Paste your prompt here\r\n```\r\n\n\n### Your Logs \ud83d\udcd2\n\n```log\r\n<insert your logs here>\r\n```\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3377/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3377/timeline", "performed_via_github_app": null, "state_reason": null}